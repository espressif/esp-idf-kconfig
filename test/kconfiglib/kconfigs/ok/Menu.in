# test if menu keyword is working
mainmenu "mainmenu"

    menu "menu with one config"

        config ONE
            bool "only config in menu"
            default y

    endmenu

    config A
        bool
        default y

    menu "menu with visible if"
        visible if A

        config B
            bool "config in menu with visible if"
            default y

    endmenu
