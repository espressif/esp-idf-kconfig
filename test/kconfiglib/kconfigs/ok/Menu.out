#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#

#
# menu with one config
#
CONFIG_ONE=y
# end of menu with one config

CONFIG_A=y

#
# menu with visible if
#
CONFIG_B=y
# end of menu with visible if
